lib/libUPnP/Platinum/Source/Devices/MediaServer/PltMediaServer.cpp

```diff
@@ -400,7 +400,8 @@ PLT_MediaServer::ParseTagList(const NPT_String& updates, NPT_Map<NPT_String,NPT_
     for (NPT_List<NPT_XmlNode*>::Iterator children = didl_partial->GetChildren().GetFirstItem(); children; children++) {
         NPT_XmlElementNode* child = (*children)->AsElementNode();
         if (!child) continue;
-        tags[child->GetTag()] = *child->GetText();
+        auto txt = child->GetText();
```

Comment:
Yes.. gettext returns a pointer that might be null. If we pass to update the playerstate an empty tag (`<xbmc:lastplayerstate></xbmc:lastplayerstate>`) we get a null from gettext. So With `*child->GetText()` we try to retrieve an object from a null pointer